### DISCUSSION TITLE
General question about creating a kata

### DISCUSSION THREAD

**Discussion by Shikiiii** [codewars/codewars.com#2409] [Q&A] [answered]
Heyo! Hope your day is going well. :D 
I have a question about making a kata, I want to make a kata about the same card game, but 2 of them with different complexity. The solutiono of the first kata won't be able to pass the second one, and it will be quite different from the other one. Would this be a duplicated kata, or is it fine?

**Comment by DonaldKellett** [comment, marked as answer]
To be safe, you should publish the easier version first and see how it goes. Then publish the harder version after the easier version proves successful.

**Reply by Shikiiii** [reply]
Alright, I will. Thank you!
Have a nice day.
